src/main/java/net/datafaker/Faker.java

```diff
@@ -1019,6 +1019,10 @@ public Zelda zelda() {
         return getProvider(Zelda.class, () -> new Zelda(this));
     }
 
+    public MassEffect massEffect() {
```

Comment:
Please follow abc ordering in providers methods
it would simplify code navigation